src/components/report-technical-issue/template.test.js

```diff
@@ -111,13 +111,13 @@ describe('Report Technical Issue', () => {
   it('should URL encode the referrer url', () => {
     const $ = render('report-technical-issue', examples['with-referrer-url']);
 
-    expect($('.govuk-link').attr('href')).toEqual('/contact/report-technical-problem?service=pay&amp;referrerUrl=https%3A%2F%2Fwww.tax.service.gov.uk%2Fpay%3Fabc%3Ddef');
+    expect($('.govuk-link').attr('href')).toEqual('/contact/report-technical-problem?service=pay&referrerUrl=https%3A%2F%2Fwww.tax.service.gov.uk%2Fpay%3Fabc%3Ddef');
```

Comment:
I reckon we should try this - so it's getting the html rather than the decoded attribute value
```suggestion
    expect($('.govuk-link').prop('outerHTML')).toContain('href="/contact/report-technical-problem?service=pay&amp;referrerUrl=https%3A%2F%2Fwww.tax.service.gov.uk%2Fpay%3Fabc%3Ddef"');
```

Reply:
I guess for both tests